exporter/collector/breaking-changes.md

```diff
@@ -26,6 +26,16 @@ domain name of the metric type.  E.g. if a metric called
 `workload.googleapis.com/nginx/latency` is created, the display name will
 be `nginx/latency` instead of `workload.googleapis.com/nginx/latency`.
 
+## Summaries
+
+The previous exporter would split summary metrics into `_summary_count`, `_summary_sum`, `_summary_percentile`, and the percentiles metric had the label `percentile` with values in 0 < 100. The new exporter splits summary metrics into `_count`, `_sum`, no suffix. The metric without a suffix has a `quantile`, with values in 0 < 1. This change was done to match the semantics of prometheus summary metrics, which users are more familiar with. The `summary_percentiles` option reverts to the previous behavior:
+
+```yaml
+googlecloud:
+  metric:
+    summary_percentiles: true
```

Comment:
Do we have first party use cases for this option, or is it just for external users who wish to keep the old behavior?

Reply:
This is just for external users.  There is one anthos metric with this naming (cc @catherinef-dev), but they seem to want the new behavior.

Reply:
Since this breaks naming of their metrics, I was thinking we would need to support the old names longer-term.  Do you think we should just change naming without a config option?

Reply:
Given other breaking changes (and that we're putting this behind a feature gate), we can break the naming here.